docs/containers.conf.5.md

```diff
@@ -29,6 +29,10 @@ Note, container engines also use other configuration files for configuring the e
 container images.
 * `policy.conf` for controlling which images can be pulled to the system.
 
+Note: If Podman is running in a virtual machine using `podman machine` (this
+includes Mac and Windows hosts), ensure that you edit the configuration files in the
```

Comment:
Nit: avoid using "you" in man pages.  In this case the you brings no value.
`ensure that the configuration files are edited in the`